### DISCUSSION TITLE
QuartoPub "page not found"

### DISCUSSION THREAD

**Discussion by cjsrussell** [quarto-dev/quarto-cli#6419] [Q&A]
### Description

I just loaded up a QuartoPub page that I hadn't touched in a couple of months. When I last used it everything worked fine but now all the links are broken? The index/homepage works fine but every time i click a link to go to a chapter/page I get the below error which directs to a Netlify help page? 

I get the same error whether using its direct url, or when previewing it on quartopub.com

![image](https://github.com/quarto-dev/quarto-cli/assets/42642025/7b6ea2ed-c14d-4e7d-8a82-ac2202886246)

I have tried entirely deleting the pub and rehosting it and have the same issue...

Any suggestions? Local preview works fine.

**Comment by isandu** [comment]
Hi there, same issue as @cjsrussell but with a website which worked fine a few hours ago. I've tried different accounts, deleted quarto and reinstalled. Checked R and quarto versions. Local preview works fine. Here is one of the websites I've tried: https://misandu.quarto.pub/carte/

**Reply by cjsrussell** [reply]
The link shared by @isandu shows exactly the problem i have - you cant navigate via the sidebar links.

Additional info: external links work fine e.g.  to my twitter/github pages

**Comment by mcanouil** [comment]
FYI, we are investigating.
Thanks for the report.

**Comment by thoughtfulbloke** [comment]
In the case of my site, https://thoughtfulnz.quarto.pub/nzcovidreport/  as I write this the navigation bar works to take people to an additional page, like https://thoughtfulnz.quarto.pub/nzcovidreport/wastewatabeta.html but the navigation bar on that page has lost the site navigation basis so would go https://thoughtfulnz.quarto.pub/wastewatabeta.

Equally if you start from https://thoughtfulnz.quarto.pub/nzcovidreport/index.html then the links are broken when they were not when starting from https://thoughtfulnz.quarto.pub/nzcovidreport/

**Comment by dragonstyle** [comment]
Quick update - we have isolated the issue and are working to resolve it. It is related to a configuration change in the way the sites are hosted and shouldn't require any changes to your published sites to begin working once we've corrected the configuration (that turns out be slightly more tricky than merely changing a setting). I don't currently have a firm ETA on the resolution but it is being worked on as I write this.

**Reply by jkylearmstrong** [reply]
Thank you very much! Please keep us in the loop!

**Comment by jkylearmstrong** [comment]
I am having this exact same issue 

I was just trying to push some changes I made, the landing page
```
https://user.quarto.pub/book/
```
looks fine, but when I try to navigate to another page with the the next page button or the chapter select I get this message:
![image](https://github.com/quarto-dev/quarto-cli/assets/29757798/f2440106-9eb0-49b4-b442-9733daf6ede3)
I am noticing is that when I try to navigate to a different page it goes to
```
https://user.quarto.pub/chapter/section
```
as opposed to
```
https://user.quarto.pub/book/chapter/section
```
If I add the book to the url link I can see my updates to the pages

**Comment by dragonstyle** [comment]
We have applied the configuration change and all affected sites should be back with links functioning normally. Newly published sites should not see any issue.

We're sorry for the bump in the road and appreciate everyone's patience while we worked through finding and deploying the fix!

**Reply by cjsrussell** [reply]
No worries at all! Thanks for sorting this so quickly! :)

**Comment by andypicke** [comment]
Thanks for the quick fix!

On Thu, Aug 3, 2023 at 8:05 PM Charles Teague ***@***.***>
wrote:

> We have applied the configuration change and all affected sites should be
> back with links functioning normally. Newly published sites should not see
> any issue.
>
> We're sorry for the bump in the road and appreciate everyone's patience
> while we worked through finding and deploying the fix!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/6419#discussioncomment-6633349>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABXIJF5UNVTKS6IHMYZQFLDXTRKFBANCNFSM6AAAAAA3CT25MU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>
-- 
Andy Pickering
***@***.***
Linkedin <https://www.linkedin.com/in/andypicke/>
Github <https://github.com/andypicke>
Blog and Website <https://andypicke.quarto.pub/portfolio/>
